backend/btrixcloud/orgs.py

```diff
@@ -501,6 +517,8 @@ async def rename_org(
         org: Organization = Depends(org_owner_dep),
     ):
         org.name = rename.name
```

Comment:
Is there any reason to make rename.name optional as well, so can change just slug but not name?

Reply:
I guess frontend can always pass just current name, so maybe not...

Reply:
I went back and forth on this myself and landed at the same place haha